reframe/core/pipeline.py

```diff
@@ -1345,6 +1348,12 @@ def run(self):
         exec_cmd = [self.job.launcher.run_command(self.job),
                     self.executable, *self.executable_opts]
         commands = [*self.prerun_cmds, ' '.join(exec_cmd), *self.postrun_cmds]
+        if type(self.build_system).__name__ == 'EasyBuild':
```

Comment:
Usually this is done with:

```python
if isinstance(self.build_system, EasyBuild):
```

(but then you need to be able to import `EasyBuild` in here, not sure if that's possible...)